test/AzureExtension/AzureExtension.Test.csproj

```diff
@@ -6,7 +6,6 @@
     <Nullable>enable</Nullable>
     <RootNamespace>AzureExtension.Test</RootNamespace>
     <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <PublishProfile>Properties\PublishProfiles\win10-$(Platform).pubxml</PublishProfile>

```

Comment:
how come this is coming out?

Reply:
Just a test project.  I didn't think it needed a publish profile.  Do you have a use for this?

Reply:
nope, just curious